core/src/main/java/org/apache/spark/util/collection/unsafe/sort/UnsafeSorterSpillReader.java

```diff
@@ -148,4 +141,34 @@ public void close() throws IOException {
      }
    }
   }
+
+  private void readFile() throws IOException {
```

Comment:
nit: How about `readFile` -> `readSpilledFile`

Reply:
I will rename method name to readSpilledFile. Thank you